pkg/module/go_get_fetcher.go

```diff
@@ -120,6 +115,38 @@ func getSources(goBinaryName string, fs afero.Fs, gopath, repoRoot, module, vers
 	return nil
 }
 
+func prepareEnv(gopath string) []string {
+	names := []string{
+		"GOPATH",
+		"GOCACHE",
+		"CGO_ENABLED",
+		"GO111MODULE",
+		"GOPROXY",
+	}
+	parentEnv := os.Environ()
+	shouldRemove := func(parentEnvVar string) bool {
+		for _, n := range names {
+			if strings.Contains(parentEnvVar, n) {
```

Comment:
why Contains? shouldnt it be `==`?

Reply:
The environment var is something like 'GOPROXY=http://something' so it won't match with ==

Reply:
oh i see. what about HasPrefix n+'='
i want to avoid false positives

Reply:
But HasPrefix would make more sense probably.

Reply:
Hehe you have been faster.